app/src/main/res/values/strings.xml

```diff
@@ -21,6 +21,10 @@
     <string name="wrong_token_message">Token invalide</string>
     <string name="empty_token_message">Token vide</string>
     <string name="connection_error_message">Erreur de connexion</string>
+
+    <string name="wrong_quiz_id_message">Ce quiz n\'existe pas</string>
+    <string name="empty_quiz_error_message">Ce quiz est vide</string>
```

Comment:
Since I merged the languages update, when you update your branch with master, you will need to put these two strings in values-fr/strings.xml and translate them in English in the values/strings.xml (the default one)

Reply:
I've put a todo as well

Reply:
I did it